.bazelrc

```diff
@@ -1 +1,6 @@
 run --workspace_status_command="bash tools/workspace-status.sh"
+
+# Don't build protoc from the cc_binary, it's slow and spammy when cache miss
+common --per_file_copt=external/.*protobuf.*@--PROTOBUF_WAS_NOT_SUPPOSED_TO_BE_BUILT
+common --host_per_file_copt=external/.*protobuf.*@--PROTOBUF_WAS_NOT_SUPPOSED_TO_BE_BUILT
```

Comment:
So I guess you're adding these lines to ensure that the build fails if protoc does end up getting built from source, right?

If so, please remove this. It ends up making this repository unbuildable on platforms for which toolchains_protoc provides no prebuilt binary.

Reply:
That's right. I think it's desirable to prevent accidental new dependencies on the protoc `cc_binary` target, which can creep in via dependencies. We could add this flag only to the CI configuration for that purpose.

Reply:
Moved to `.github/workflows/ci.bazelrc` which gets copied to the home directory's bazelrc during bazel install. This way we are sure all CI actions use that flag.